src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -948,341 +946,25 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
     }));
     //#endregion
 
-    //#region gesture methods
-    const handleGestureStart = useWorkletCallback(
```

Comment:
I just moved all of this over to src/components/bottomSheet/useInteractivePanGestureHandlerListeners.ts